### ISSUE TITLE
Docs regarding named ranges or lists incorrect?

### ISSUE THREAD

**Issue by preed** [cronie-crond/cronie#62]
I believe the following verbiage in `man/crontab.5` is incorrect:

```
Names can also be used for the 'month' and 'day of week' fields.  Use the
first three letters of the particular day or month (case does not
matter).  Ranges or lists of names are not allowed.
```

I perused the source code, and based on my reading (which you shouldn't trust :-) believe that the code indeed does convert weekday and month names into integers, and then uses the integers to trigger on the appropriate range (or list) of days (or months); see: https://github.com/cronie-crond/cronie/blob/master/src/entry.c#L248-L264 and https://github.com/cronie-crond/cronie/blob/master/src/entry.c#L438

I also just ran a test (just to be sure my reading of the source wasn't entirely off) using the follow crontab entries, and have been getting emails as expected all week:
```
0 12 * * mon-fri echo "Cronie day range test: $(date +\%a)"
0 12 * * sat,sun echo "Cronie day list test: $(date +\%a)"
```

I didn't try named month lists or ranges, but looking at the code, it uses very similar logic for month-names, so it seems like it should also work.

I suggest the following patch:

```
diff --git a/man/crontab.5 b/man/crontab.5
index 2ee9feb..abe2b1c 100644
--- a/man/crontab.5
+++ b/man/crontab.5
@@ -212,7 +212,8 @@ two hours, you can use "*/2".
 .PP
 Names can also be used for the 'month' and 'day of week' fields.  Use the
 first three letters of the particular day or month (case does not
-matter).  Ranges or lists of names are not allowed.
+matter).  Ranges and lists of names are allowed. Examples: "mon,wed,fri",
+"jan-mar".
 .PP
 If the UID of the owner is 0 (root), the first character of a crontab
 entry can be "-" character. This will prevent cron from writing a syslog
```

**Comment by t8m** [comment]
Would you like to submit the patch as pull request?

**Comment by t8m** [comment]
Fixed by #63
